### ISSUE TITLE
Send your topic for MLoN Workshop on May 18 and 19

### ISSUE THREAD

**Issue by hshokrig** [hshokrig/EP3260-MLoNs-2020#14]
Our MLoN workshop is on May 18 and 19, and so far we have Prof. Poor from Princeton as a keynote speaker. 

Each Study Group (SG) is supposed to prepare a 45 minutes oral presentation, followed by 15 minutes Q&A. Due to the COVID outbreak, we have canceled the poster part.

In the following, I provided some potential topics. But you are welcome to suggest new topics as well. You should choose some prominent papers (does not have to be new or the ones we suggested) to cover various aspects of that topic. Send me ASAP a list of three topics for your group (with your priority ranks). I will assign the topics to you afterward and your scheduled presentation time. **ALL MEMBERS SHOULD PRESENT IN THE VIRTUAL WORKSHOP DAYS IF YOU WANT THE COURSE CREDIT.** You should inform me if you cannot attend. 

**Special Topic 1: Application areas: Federated learning and privacy-preserving distributed MLoN**

1.  Konečný, H. B. McMahan, D. Ramage, P. Richtárik, “Federated optimization: Distributed machine learning for on-device intelligence”. arXiv preprint arXiv:1610.02527, 2016.
2. T. Li, A. K. Sahu, M. Zaheer, M. Sanjabi, A. Talwalkar, and V. Smith, “Federated Optimization in Heterogeneous Networks,” arXiv preprint arXiv:1812.06127, Apr. 2020.
3. P. Kairouz, H. B. McMahan, B. Avent, A. Bellet, M. Bennis, et al., “Advances and open problems in federated learning,” arXiv preprint arXiv:1912.04977, Dec. 2019.
4. S. Truex, N. Baracaldo, A. Anwar, T. Steinke, H. Ludwig, R. Zhang, and Y. Zhou, “A Hybrid Approach to Privacy-Preserving Federated Learning”, In Proceedings of the 12th ACM Workshop on Artificial Intelligence and Security (AISec’19), ACM, New York, NY, USA, 1–11, 2019. DOI: https://doi.org/10.1145/3338501.3357370

**Special Topic 2: Handling Non-IID datasets**

1. Zhao, Yue, et al. "Federated learning with non-iid data." arXiv preprint arXiv:1806.00582 (2018).
2. Sattler, Felix, et al. "Robust and communication-efficient federated learning from non-iid data." IEEE transactions on neural networks and learning systems (2019).
3. Li, Xiang, et al. "On the convergence of fedavg on non-iid data." arXiv preprint arXiv:1907.02189 (2019).

**Special Topic 3: Model Compression**

1. Cheng, Yu, et al. "A survey of model compression and acceleration for deep neural networks." arXiv preprint arXiv:1710.09282 (2017).
2. Han, Song, Huizi Mao, and William J. Dally. "Deep compression: Compressing deep neural networks with pruning, trained quantization and huffman coding." arXiv preprint arXiv:1510.00149 (2015).
3. Iandola, Forrest N., et al. "SqueezeNet: AlexNet-level accuracy with 50x fewer parameters and< 0.5 MB model size." arXiv preprint arXiv:1602.07360 (2016).

**Special Topic 4: Error compensation and variance reduction**

1. tich, Sebastian U., and Sai Praneeth Karimireddy. "The error-feedback framework: Better rates for SGD with delayed gradients and compressed communication." arXiv preprint arXiv:1909.05350 (2019).
2. Horváth, Samuel, et al. "Stochastic distributed learning with gradient quantization and variance reduction." arXiv preprint arXiv:1904.05115 (2019).

**Comment by mairtonbjr** [comment]
Based on the topics and priority ranks sent by each SG, we decided on the following assignment:

Group 1 - ST 3
Group 2 - ST 2 and ST 4
Group 3 - ST 1
Group 4 - ST 3

Please send the papers of your presentation ASAP to our emails or as replies to this issue.

**Comment by YIN95** [comment]
[A Survey of Model Compression and Acceleration for Deep Neural Networks.pdf](https://github.com/hshokrig/EP3260-MLoNs-2020/files/4605715/A.Survey.of.Model.Compression.and.Acceleration.for.Deep.Neural.Networks.pdf)
[Deep Compression- Compressing Deep Neural Networks with Pruning, Trained Quantization and Huffman Coding .pdf](https://github.com/hshokrig/EP3260-MLoNs-2020/files/4605717/Deep.Compression-.Compressing.Deep.Neural.Networks.with.Pruning.Trained.Quantization.and.Huffman.Coding.pdf)
[MobileNetV2- Inverted Residuals and Linear Bottlenecks.pdf](https://github.com/hshokrig/EP3260-MLoNs-2020/files/4605718/MobileNetV2-.Inverted.Residuals.and.Linear.Bottlenecks.pdf)
[Recent Advances in Efficient Computation of Deep Convolutional Neural Networks.pdf](https://github.com/hshokrig/EP3260-MLoNs-2020/files/4605719/Recent.Advances.in.Efficient.Computation.of.Deep.Convolutional.Neural.Networks.pdf)

Hi, 

The papers of our presentation are attached. 

Best regards,
Group 1

**Comment by henrikhellstrom93** [comment]
Hello,

Following are the papers for our presentation:

[A Tight Convergence Analysis for Stochastic Gradient Descent with Delayed Updates
](https://arxiv.org/abs/1806.10188)
[Stochastic Distributed Learning with Gradient Quantization and Variance Reduction
](https://arxiv.org/abs/1904.05115)
[The Error-Feedback Framework: Better Rates for SGD with Delayed Gradients and Compressed Communication
](https://arxiv.org/abs/1909.05350)
[DOUBLESQUEEZE: Parallel Stochastic Gradient Descent with Double-pass
Error-Compensated Compression](https://arxiv.org/abs/1905.05957)

Bests,
Group 4
